manual/python/index.md

```diff
@@ -87,9 +173,53 @@ root [2] TPython::Prompt()
 
 For a more complete description of the `TPython` interface, please check the reference guide for {% include ref class="TPython" %}.
 
-## JupyROOT: PyROOT for Jupyter notebooks
+## JupyROOT: (Py)ROOT for Jupyter notebooks
+
+[Jupyter notebooks](https://jupyter.org/) provide a web-based interface that can be used to do interactive computing with ROOT.
+
+### How to use it
+
+Two kernels (or language flavours) allow to run ROOT from Jupyter: *Python* (2 and 3) and *C++*. In order to use such kernels, there are mainly two options:
+
+- Local ROOT installation: to install ROOT locally, please follow the instructions [here]({{ '/install' | relative_url }}). In addition to ROOT, two Python packages are required, which can be installed e.g. with `pip`:
+
+```shell
+pip install jupyter
+pip install metakernel
+```
+
+Once all requirements are installed, a Jupyter server with the Python and C++ kernels can be started by running from a terminal:
```

Comment:
How is `root --notebook` any different from `jupyter notebook`? I can also access the cpp kernel from there. I've never experienced any advantage of `root --notebook` compared to `jupyter notebook`. In case there is any difference, it might be worth to point them out here.

Reply:
`root --notebook` installs the ROOT C++ kernel in the home directory of the user (if not there already) and sets the environment so that Jupyter finds it. If you run `jupyter notebook` but you have never run `root --notebook` before, you should not see the ROOT C++ kernel unless you install it manually. That is why here I talk about `root --notebook`.